src/features/todos.ts

```diff
@@ -1,9 +1,47 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetTodos = { type: 'set', payload: Todo[] };
+type LoadTodos = { type: 'load' };
 
-const todosReducer = (): Todo[] => {
-  return [];
+type Actions = SetTodos | LoadTodos;
+
+const set = (todosFromServer: Todo[]): SetTodos => (
+  { type: 'set', payload: todosFromServer }
+);
+
+const load = (): LoadTodos => (
+  { type: 'load' }
+);
+
+type InitialState = {
+  todos: Todo[],
+  load: boolean,
+};
+
+const initialState: InitialState = {
+  todos: [],
+  load: false,
+};
+
+// eslint-disable-next-line max-len
+const todosReducer = (state: InitialState = initialState, action: Actions): InitialState => {
```

Comment:
```suggestion
const todosReducer = (
  state: InitialState = initialState, 
  action: Actions
): InitialState => {
```
Don't disable the eslint until it's really necessary